### ISSUE TITLE
cuda error driver 719

### ISSUE THREAD

**Issue by olegaix** [nicehash/excavator#77]
53:48][0x000000b8][info] net | Share #12 accepted
[19:53:52][0x000000b8][info] core | Algorithm 'equihash' total speed: 2.442794 kH/s
[19:53:53][0x000000b8][info] net | Share #13 accepted
[19:53:54][0x0000201c][info] net | Share #14 accepted
[19:54:03][0x000000b8][info] net | Share #15 accepted
[19:54:04][0x0000201c][info] net | Share #16 accepted
[19:54:07][0x000000b8][info] core | Algorithm 'equihash' total speed: 2.435753 kH/s
[19:54:15][0x00001c88][fatal] wrkr3-3 | CUDA error DRIVER: '719' in func 'run_single_stream' line 960
[19:54:15][0x0000103c][fatal] wrkr6-6 | CUDA error DRIVER: '719' in func 'run_single_stream' line 960
[19:54:15][0x00000c84][fatal] wrkr0-0 | CUDA error DRIVER: '719' in func 'run_single_stream' line 960
[19:54:15][0x00001a44][fatal] wrkr2-2 | CUDA error DRIVER: '719' in func 'run_single_stream' line 960
[19:54:15][0x000008d0][info] Shutting down
[19:54:15][0x00000438][fatal] wrkr7-7 | CUDA error DRIVER: '719' in func 'run_single_stream' line 960
[19:54:15][0x00001f84][fatal] wrkr1-1 | CUDA error DRIVER: '719' in func 'run_single_stream' line 960
[19:54:15][0x0000149c][fatal] wrkr5-5 | CUDA error DRIVER: '719' in func 'run_single_stream' line 960
[19:54:15][0x00001ff0][fatal] wrkr4-4 | CUDA error DRIVER: '719' in func 'run_single_stream' line 960
[19:54:16][0x000008d0][info] http | Closing
Excavator has crashed... restarting in 10 seconds...

after 20 seconds of runing it goes ito error driver 719 why?

**Comment by dropky** [comment]
The driver probably crashed due to overclocking/power issues.

**Comment by Konz1** [comment]
This thread is a little aged now, but still happened to me. I wanted to share my experience on this. I spent days trying to figure this CUDA driver error 719 / 702 stuff. I tried EVERYTHING: 
- Nvidia drivers new and old
- swapping cables
- different versions of NiceHash
- moving my MSI 1070's around to different PCIe slots.
- tried cranking down my GPU's to less power/clocks
- different mining algorithms in NiceHash (I suspected 1 GPU had a problem with Equihash)
- flashing my motherboard BIOS
- looked for conflicting drivers/apps

I was almost ready to give up, knowing my rig was running fine when i had GPU's and then with my 3rd and 4th this began to fail. I was about to start assuming I had defective graphic cars.

And the final trick that worked?  
Turns out I had a bad PCIe slot #3. So once I moved my connections away from it and used other slots, ALL IS RUNNING PERFECTLY!!! Whew! Finally!
